browser/templates/browser/person.html

```diff
@@ -2,12 +2,48 @@
 {% load app_filters %}
 {% block title %}{{ person.name }} (Person){% endblock %}
 
-
+{% load staticfiles %}
 {% block main %}
-
 <script>
         $(document).ready(function() {
+
+            var person_url_params = window.location.href.split("?");
+            if(typeof person_url_params[1] !== 'undefined' && typeof person_url_params[1] !== 'null' )
+            {
+                $('.nav-tabs a[href="#' + person_url_params[1] + '"]').tab('show');
+            }
```

Comment:
if you use proper parameters in the url, you can simply get a parameter from the request using django's template syntax.